addons/hide-project-stats/userscript.js

```diff
@@ -0,0 +1,73 @@
+export default async function ({ addon, global, console, msg, safeMsg: m }) {
+  let loves = { name: "love" };
+  let favorites = { name: "favorite" };
+  await addon.tab.waitForElement(".project-loves");
+  let visitingOwnProject = document.querySelector(".button.action-button.report-button") == null;
+
+  function initializeLabels(button) {
+    button.buttonElement = document.querySelector(`.project-${button.name}s`);
+    button.labelElement = document.createElement("span");
+    button.labelElement.id = `sa-${button.name}-label`;
+    button.buttonElement.after(button.labelElement);
+  }
+
+  function refreshLabels() {
+    refreshButton(loves);
+    refreshButton(favorites);
+    // If the user has the "show stats on my own projects" setting enabled,
+    // show the remix and view count if the user owns the project.
+    // Love and favorite counts are handled separately since they get custom labels.
+    document.querySelectorAll(".project-remixes, .project-views").forEach((element) => {
+      if (!addon.self.disabled && addon.settings.get("showOwnStats") && visitingOwnProject) {
+        element.style = "display: flex;";
```

Comment:
use element.style.display

Reply:
Are you sure that works? Last I checked, doing it that way is depreciated...

Reply:
Add a class and use userstyle.

Reply:
`element.style.display = "flex"` and `element.style = "display: flex;"` are equivalent (well, and the second one is worse, because it removes any existing inline styles)